test.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-
+python -m pytest test_challenger.py 
```

Comment:
I think I should have pushed a separate PR first to add `test_challenger.py`. I will do that.

Reply:
Yes, please